search-provider/eks-discovery-feed-provider.c

```diff
@@ -682,6 +691,9 @@ artwork_card_descriptions_cb (GObject *source,
           add_key_value_pair_from_model_to_variant (model, &builder, "synopsis");
         }
 
+      /* Start building up object */
+      g_variant_builder_open (&builder, G_VARIANT_TYPE ("a{ss}"));
```

Comment:
This is needed for the block above, since we call `add_key_value_pair_to_variant()`, but you moved it all the way down here...

Reply:
You're right, not sure why I moved it down that far ...